main.go

```diff
@@ -27,6 +27,7 @@ import (
 	"net/url"
 	"os"
 	"os/signal"
+	pathmodule "path"
```

Comment:
any reason we cannot leave this as `"path"`?

Reply:
because "path" is already used as a symbol for variable in the scope, I rename the module import to avoid confusing the compiler.

Reply:
I think it is reasonable to rename the local variable to `p`. The usage distance of that variable is bound to the for loop.

Reply:
ok, i will rename local variable 👍